### PR TITLE
Revert Changes before transitioning

### PR DESCRIPTION
**Pull request by colemanProjects**
While upgrading my project which depends on navi. I get the error: 
Assertion Failed: Attempted to call .send() with the action 'revertChanges' on the destroyed route 'reports.report'

Found that in ember 3.2 they've added an assertion to check for actions being sent on destroyed components.
https://github.com/emberjs/ember.js/issues/16820

speaks of a workaround

fixes https://github.com/yahoo/navi/issues/287

### CODE REVIEW

**Comment by colemanProjects** [comment]
tested in my app, causes my tests to pass.

File: packages/reports/tests/acceptance/navi-report-test.js [hunk 1]
```diff
@@ -154,14 +154,63 @@ test('New report - copy api', function(assert) {
...
+  andThen(() => {
+    assert.ok(
+      $('.checkbox-selector--dimension .grouped-list__item:contains(Day) .grouped-list__item-label input').is(
+        ':checked'
+      ),
+      'Day timegrain is checked by default'
+    );
+  });
+
+  // Remove uncheck the day timegrain
```
Comment (jkusa) [inline packages/reports/tests/acceptance/navi-report-test.js]: Uncheck

File: packages/reports/tests/acceptance/navi-report-test.js [hunk 2]
```diff
@@ -154,14 +154,63 @@ test('New report - copy api', function(assert) {
...
+    );
+    assert.ok(
+      $('.navi-report__revert-btn').is(':visible'),
+      'Revert changes button is visible once a change has been made'
+    );
+  });
+
+  // leave the route
+  visit('/reports');
+  andThen(() => {
```
Comment (jkusa) [inline packages/reports/tests/acceptance/navi-report-test.js]: Can we remove this

File: packages/reports/tests/acceptance/navi-report-test.js [hunk 3]
```diff
@@ -154,14 +154,63 @@ test('New report - copy api', function(assert) {
...
+    // enter the route again
+    click("a[href$='/reports/1/view']");
+    andThen(() => {
+      assert.notOk(
+        $('.navi-report__revert-btn').is(':visible'),
+        'After navigating away and back to the route, the Revert button disappears and changes are reverted'
```
Comment (jkusa) [inline packages/reports/tests/acceptance/navi-report-test.js]: `'After navigating away and back to the route, the revert button disappears`

**Pull request merged**